scripts/analysis.py

```diff
@@ -638,50 +583,171 @@ def final_stockpile(cursor, facility):
     return outstring
 
 
-    return outstring
+def fuel_into_reactors(cursor):
+    """ Finds timeseries of mass of fuel received by reactors
 
- 
-def fuel_usage_timeseries(cursor, fuel_list):
-    """ Calculates total fuel usage over time
+    Parameters
+    ----------
+    cursor: sqlite cursor
+        sqlite cursor
+
+    Returns
+    -------
+    get_timeseries: function
```

Comment:
My bad there, I wasn't sure which one I should write (a function call, or what that function returns).
Thanks for the clarification.